fec.config.js

```diff
@@ -13,4 +13,15 @@ module.exports = {
    * Add additional webpack plugins
    */
   plugins: [],
+  moduleFederation: {
+    exposes: {
+      './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      './LearningResourcesWidget': path.resolve(__dirname, './src/components/LearningResourcesWidget/LearningResourcesWdiget.tsx'),
```

Comment:
We probably don't want to expose the widget until we agree on the common approach - https://redhat-internal.slack.com/archives/C05M0SNTLM8/p1710170397998599